app/services/report_service.rb

```diff
@@ -44,11 +44,11 @@ def payload
       @report,
       serializer: ActivityPub::FlagSerializer,
       adapter: ActivityPub::Adapter,
-      account: some_local_account
+      account: local_admin_account
     ).as_json)
   end
 
-  def some_local_account
-    @some_local_account ||= Account.local.where(suspended: false).first
+  def local_admin_account
+    @local_admin_account ||= User.admins.first.account
```

Comment:
Idea: Why not use "contact account" when it's set? Also idea: Abstract it out into, for example, `Account.representative`... :thinking:

Reply:
As in `Settings.site_contact_username` ? 

I didn't know it existed, but we certainly could use that.

Reply:
@Gargron I started doing this, and the I realised: there's absolutely no validation on the Setting.site_contact_username value — you can set it to pretty much anything.

Interestingly the about page doesn't show the correct user is site_contact_username is a remote user, despite this being accepted in settings.

Reply:
So, we'd end up with probably this:

```ruby
  def representative
    return nil unless local?

    site_contact = Account.find_local(Setting.site_contact_username) if Setting.site_contact_username.present?
    site_contact || User.admins.first.account
  end
```

If we're to follow the lead from InstancePresenter, then:
```ruby
  def representative
    return nil unless local?

    site_contact = Account.find_local(Setting.site_contact_username)
    site_contact || User.admins.first.account
  end
```

Which relies on the fact that username can't be an empty string